### ISSUE TITLE
Larry And The Gnomes: Can't start the Story.

### ISSUE THREAD

**Issue by p0008874** [ruffle-rs/ruffle#10261]
### Describe the bug

https://user-images.githubusercontent.com/75534590/226187345-00c278a9-9271-4620-bd71-8bcb3b97ba02.mp4

```
2023-03-19T15:39:57.970578Z  INFO ruffle_core::player: Loaded SWF version 10, resolution 640x360 @ 30 FPS
2023-03-19T15:39:59.120036Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.system.Security.allowDomain()
2023-03-19T15:39:59.120293Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.system.Security.allowInsecureDomain()
2023-03-19T15:40:00.528967Z  INFO avm_trace: [NewgroundsAPI] :: You have successfully connected to the Newgrounds API Gateway
2023-03-19T15:40:00.529638Z  INFO avm_trace: [NewgroundsAPI] :: Movie identified as "Larry And The Gnomes"
2023-03-19T15:40:00.532501Z  INFO avm_trace: [NewgroundsAPI] :: This movie has been approved to run Flash Ads!
2023-03-19T15:40:22.659578Z  WARN isahc::handler: request completed with error: failed to connect to the server
2023-03-19T15:40:22.700231Z  INFO avm_trace: [NewgroundsAPI] :: Medals loaded!
2023-03-19T15:40:22.705082Z ERROR ruffle_core::loader: Error during movie loading: FetchError("failed to connect to the server")
2023-03-19T15:40:23.172786Z  INFO avm_trace: [NewgroundsAPI] :: Medals loaded!
2023-03-19T15:40:23.173040Z  INFO avm_trace: [NewgroundsAPI] :: Metadata loaded!
2023-03-19T15:40:27.279112Z  WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.media.Sound constructor with arguments
2023-03-19T15:40:34.929636Z  INFO ruffle_core::focus_tracker: Focus is now on Some(Stage(Stage { ptr: 0x1ba3d83f7d0 }))
2023-03-19T15:40:56.466392Z ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::activation: AVM2 error: Object(ErrorObject(ErrorObject { class: TypeError, ptr: 0x1ba63b3a600 }))
2023-03-19T15:40:56.466808Z ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::activation: TypeError: Type Coercion failed: cannot convert Object(StageObject(StageObject { name: Main_OpeningSwf, ptr: 0x1ba64401a30 })) to flash.display.LoaderInfo.
2023-03-19T15:40:56.467517Z ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "complete", class: flash.events::Event, ptr: 0x1ba643c3ce0 }) to handler FunctionObject(FunctionObject { ptr: 0x1ba4ce285c0 }) : AvmError(Object(ErrorObject(ErrorObject { class: TypeError, ptr: 0x1ba63b3a600 })))
```

### Expected behavior

https://user-images.githubusercontent.com/75534590/226187383-2899348c-0efe-4af2-a625-a114d3e08410.mp4

### Affected platform

Desktop app

### Operating system

Windows 10

### Additional information

https://www.newgrounds.com/portal/view/538572

**Comment by Lord-McSweeney** [comment]
This needs a call handler for `LoaderInfo`- it does `LoaderInfo(some stage obj)`.

**Comment by Lord-McSweeney** [comment]
Fixed by #12553.
